### PR TITLE
palce '.' at the head of the method in method chain

### PR DESCRIPTION
**Pull request by reedfeng**
Can you consider this suggestion, it looks somewhat strange to me the '.' not before the method

### CODE REVIEW

**Comment by YeeMo** [comment]
when I first see that in a PullRequest(poor editor), i had thought it was something like a block within a context. especially after you experienced CoffeeJs

context.
  doSomething.
  doSomehing.

**Comment by AaronV** [comment]
I prefer periods at the start of the lines as well, I find it easier to read.

File: style/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Formatting
 * Indent continued lines two spaces.
 * Indent private methods equal to public methods.
 * If you break up a chain of method invocations, keep each method invocation on
-  its own line. Place the `.` at the end of each line, except the last.
+  its own line. Place the `.` at the head of each line.
   [Example][dot guideline example].
```
Comment (AaronV) [inline style/README.md]: Should probably remove this example while you're here. It doesn't link to anywhere.

**Comment by joelind** [comment]
Could somebody please close this PR, or remove the comment that mentions me? Thanks!

**Pull request closed without merging**